hipcub/include/hipcub/cub/util_allocator.hpp

```diff
@@ -0,0 +1,100 @@
+/******************************************************************************
+ * Copyright (c) 2011, Duane Merrill.  All rights reserved.
+ * Copyright (c) 2011-2018, NVIDIA CORPORATION.  All rights reserved.
+ *
```

Comment:
Modifications copyright AMD 2019 is needed here. I

Reply:
I was the reviewer on our side. Sorry for that.

Reply:
Updated.